TravelAgency-API/src/main/java/cz/muni/fi/travelAgency/DTO/CustomerCreateDTO.java

```diff
@@ -13,6 +13,8 @@
  */
 public class CustomerCreateDTO {
 
+    private Long id;
```

Comment:
Edit the annotation to be in sync with the Persistance layer annotation - @Nonnull Email, nullable number and date of birth...